homeassistant/components/ambient_station/__init__.py

```diff
@@ -73,11 +70,13 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
             Websocket(entry.data[CONF_APP_KEY], entry.data[CONF_API_KEY]),
         )
         hass.loop.create_task(ambient.ws_connect())
-        hass.data[DOMAIN][entry.entry_id] = ambient
     except WebsocketError as err:
```

Comment:
This error cannot happen because you're not awaiting the `ws_connect` coroutine. Instead, you're creating a task which means it's executed in the background.